### PR TITLE
tasks: background fixity checker

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

**Comment by lnielsen** [comment]
Depends on inveniosoftware/invenio-admin#17

**Comment by drjova** [comment]
@lnielsen from models ``serialize`` is missing. Are we gonna use the ``serializer``s from https://github.com/inveniosoftware/invenio-files-rest/pull/28?

File: invenio_files_rest/tasks.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015, 2016 CERN.
```
Comment (drjova) [inline invenio_files_rest/tasks.py]: only 2016?

File: invenio_files_rest/admin.py [hunk 2]
```diff
@@ -40,33 +47,35 @@ def require_slug(form, field):
...
+    )
+    column_details_list = (
+        'name', 'uri', 'default', 'created', 'updated', 'buckets')
+    column_list = ('name', 'uri', 'default', 'created', 'updated', 'buckets')
     column_labels = dict(
         id=_('ID'),
         uri=_('URI'),
     )
-    column_filters = ('default', 'created', 'updated', )
+    column_filters = ('default', 'created', 'updated')
```
Comment (drjova) [inline invenio_files_rest/admin.py]: shall we keep the comma?

**Comment by lnielsen** [comment]
@drjova Serialisers will be rewritten using the same logic as what has just been implemented in Records-REST.

**Comment by drjova** [comment]
@lnielsen ok! the only problem is that the example's API is broken :/ Is it possible to add dummy `serialisers` for now WDYT? Otherwise is good to go :)

**Comment by lnielsen** [comment]
@drjova Yes, #28 is adding some dummy serialisers so we can merge that one right after.

**Comment by drjova** [comment]
:shipit:

**Pull request merged**